documentation/assemblies/cruise-control/assembly-cruise-control-concepts.adoc

```diff
@@ -5,17 +5,21 @@
 [id='cruise-control-concepts-{context}']
 = Using Cruise Control for cluster rebalancing
 
-include::../../modules/cruise-control/con-cruise-control-description.adoc[leveloffset=+1]
+[role="_abstract"]
+Cruise Control is an open-source system for Kafka that monitors broker loads and rebalances partitions to optimize use of resources across the cluster.
+Rebalances help with running a more balanced Kafka cluster that uses brokers more efficiently.
```

Comment:
Since we use Cruise Control for more than just rebalances, for example topic replication factor changes, I would suggest we keep the original line here

```suggestion
The operations help with running a more balanced Kafka cluster that uses broker pods more efficiently.
```